recursividade/3-contador-regressivo/contadorRegressivo.js

```diff
@@ -0,0 +1,10 @@
+function countdown(num) {
+  if (num === 0) {
+    console.log('fim')
+  } else if (num !== 0) {
+    console.log(num)
+    countdown(num - 1)
+  }
+}
+
+console.log(countdown(10))
```

Comment:
```suggestion
countdown(10)
```

Remover esse console.log